src/cli/node/json_rpc_client.rs

```diff
@@ -463,10 +464,18 @@ pub fn send_vtt(
         None => UtxoSelectionStrategy::Random,
     };
 
+    let (fee, weighted_fee) = match (fee, weighted_fee) {
+        (Some(x), None) => (x, false),
+        (None, Some(x)) => (x, true),
+        (None, None) => bail!("Any fee included"),
```

Comment:
Maybe something clearer :)
```suggestion
        (None, None) => bail!("A fee should be specified"),
```